components/x-gift-article/src/Message.jsx

```diff
@@ -151,24 +153,31 @@ export default ({
 				<div className="x-gift-article-message x-gift-article-message--enterprise">
 					<h4>Your organisation has run out of share credits.</h4>
 					<p>
-						Request more credits and our Enterprise team will get in touch with the admin of your FT
-						subscription to arrange a top-up of sharing credits.
+						Request more credits and our team will get in touch with the admin of your FT subscription to
+						arrange a top-up of sharing credits.
 					</p>
 					<a
-						href="https://enterprise.ft.com/ft-enterprise-sharing-request-access/?segmentId=c87259e0-7073-3ea8-7f83-b988f05c3f94"
-						target="_blank"
+						href="mailto:customer.success@ft.com"
```

Comment:
Thought: I would rather not use an e-mail directly. However, this part should be visible only to B2B users, is that right?

Reply:
The requirement of the ticket in this case is to include a `mailto` link directly ([ENTST-153](https://financialtimes.atlassian.net/jira/software/projects/ENTST/boards/1569?selectedIssue=ENTST-153)) but yes, this button will only be visible to B2B users and only when they have run out of sharing credits